features/56-login-with-custom-token/api.md

```diff
@@ -0,0 +1,160 @@
+## Feature Overview
+
+Allow user to integrate their authentication server to skygear server by logging in with custom token
+
+## Guides
+
```

Comment:
Please add the `Scenario`. i.e. Developer already have his own user system, but he want to get one of Skygear plugin, chat for example.